am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/AccessManagementServiceIntegrationTest.java

```diff
@@ -43,10 +45,40 @@ public void createQuery_whenCreatingResourceAccess_ResourceAccessAppearsInDataba
         ams.createResourceAccess(createRecord(resourceId, ACCESSOR_ID, EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS));
 
         int count = jdbi.open().createQuery(
-                "select count(1) from access_management where resource_id = ?")
-                .bind(0, resourceId)
-                .mapTo(int.class)
-                .findOnly();
+            "select count(1) from access_management where resource_id = ?")
+            .bind(0, resourceId)
+            .mapTo(int.class)
+            .findOnly();
+
+        assertThat(count).isEqualTo(1);
+    }
+
+    @Test
+    public void revokeResourceAccess_whenRevokingResourceAccess_ResourceAccessRemovedFromDatabase() {
```

Comment:
May I suggest having test class per method? When class have more then few methods I find it difficult to find tests that target method under change despite the fact that test methods are prepended with name of tested method.